pkg/utils/files.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func FindFiles(humanOutput bool, args ...string) ([]string, error) {
+	var files []string
+	for _, fileOrDir := range args {
+		if info, err := os.Stat(fileOrDir); err != nil {
+			return nil, err
+		} else if info.IsDir() {
+			if entries, err := os.ReadDir(fileOrDir); err != nil {
+				return nil, err
+			} else {
+				for _, entry := range entries {
+					if sub, err := FindFiles(humanOutput, filepath.Join(fileOrDir, entry.Name())); err != nil {
+						return nil, err
+					} else {
+						files = append(files, sub...)
+					}
+				}
+			}
+		} else {
+			ext := strings.ToLower(filepath.Ext(fileOrDir))
+			if ext == ".json" || ext == ".yaml" || ext == ".yml" {
+				files = append(files, fileOrDir)
```

Comment:
What's the point ?

Reply:
Find files that are not named with the given extension. I could see people add some `kustomize` schema in the schema (since it has patching ability, that's totally an option), and kustomize files are typically named `Kustomize` I think (no extension).

I'm not 100% sure on this one, let's forget it for now, we can come back to it if it becomes a requirement.

Reply:
if i'm not mistaken kustomize uses `kustomization.yaml` file name.

Reply:
Then that'd be perfect, thanks! :-)